src/PKSim.Infrastructure/ProjectConverter/v10/Converter9To10.cs

```diff
@@ -63,6 +65,17 @@ private void convertIndividualProtein(XElement proteinNode)
          _converted = true;
       }
 
+      private void convertIndividualTransporter(XElement transporterNode)
+      {
+         transporterNode.Descendants("TransporterExpressionContainer").Each(transporterContainerNode =>
+         {
+            // membraneLocation is not used anymore. We will save it temporarily as Tag so that we can access it during object conversion
+            var membraneLocation = EnumHelper.ParseValue<MembraneLocation>(transporterContainerNode.GetAttribute("membraneLocation"));
+            var tag = new XElement("Tag");
+            tag.SetAttributeValue("value", membraneLocation);
```

Comment:
@Yuri05 Little bit of a hack here.... because membraneLocation is not serialized anymore, I need to take it from the XML and save it temp in a property that I will be able to access in my object. We have tags so I am using it. Adding the location as tag..